HPXMLtoOpenStudio/resources/utility_bills.rb

```diff
@@ -105,47 +47,39 @@ def self.marginal_rate_to_average_rate(marginal_rate, fixed_charge, household_co
     return marginal_rate + 12.0 * fixed_charge / household_consumption
   end
 
-  def self.get_state_code_to_padd
-    # https://www.eia.gov/tools/glossary/index.php?id=petroleum%20administration%20for%20defense%20district
-    padd_to_state_codes = { 'PADD 1A' => ['CT', 'MA', 'ME', 'NH', 'RI', 'VT'],
-                            'PADD 1B' => ['DE', 'DC', 'MD', 'NJ', 'NY', 'PA'],
-                            'PADD 1C' => ['FL', 'GA', 'NC', 'SC', 'WV', 'VA'],
-                            'PADD 2' => ['IA', 'IL', 'IN', 'KS', 'KY', 'MI', 'MN', 'MO', 'ND', 'NE', 'OH', 'OK', 'SD', 'TN', 'WI'],
-                            'PADD 3' => ['AL', 'AR', 'LA', 'MS', 'NM', 'TX'],
-                            'PADD 4' => ['CO', 'ID', 'MT', 'UT', 'WY'],
-                            'PADD 5' => ['AK', 'AZ', 'CA', 'HI', 'NV', 'OR', 'WA'] }
+  def self.get_eia_seds_rate(state_code, fuel_type)
+    msn_code_map = {
+      HPXML::FuelTypeElectricity => 'ESRCD',
+      HPXML::FuelTypeNaturalGas => 'NGRCD',
+      HPXML::FuelTypeOil => 'DFRCD',
+      HPXML::FuelTypePropane => 'PQRCD',
+      HPXML::FuelTypeCoal => 'CLRCD',
+      HPXML::FuelTypeWoodCord => 'WDRCD',
+      HPXML::FuelTypeWoodPellets => 'WDRCD'
+    }
+    unit_conv = {
+      HPXML::FuelTypeElectricity => 0.003412, # convert $/MMBtu to $/kWh
+      HPXML::FuelTypeNaturalGas => 0.1, # convert $/MMBtu to $/therms
+      HPXML::FuelTypeOil => 0.139, # convert $/MMBtu to $/gallons of oil
+      HPXML::FuelTypePropane => 0.0916, # convert $/MMBtu to $/gallons of propane
+      HPXML::FuelTypeCoal => 1, # $/MMBtu
+      HPXML::FuelTypeWoodCord => 1, # $/MMBtu
+      HPXML::FuelTypeWoodPellets => 1 # $/MMBtu
```

Comment:
Can you use `UnitConversions.convert(...)` here? That way we use consistent unit conversions everywhere. Also, just use MBtu, not MMBtu. We don't use the MMBtu convention anywhere and it will be confusing if we started to use it some places.

For the gal <-> MBtu conversion, since it's not strictly a unit conversion, I suggest adding a new method to the UnitConversions class to handle it.